File: src/main/java/de/akquinet/ccsp/floodmark/repository/MeasurementImportRepository.java
package de.akquinet.ccsp.floodmark.repository;

import de.akquinet.ccsp.floodmark.datamodel.MeasurementImport;
import java.util.Date;
import org.springframework.data.mongodb.repository.MongoRepository;

public interface MeasurementImportRepository extends MongoRepository<MeasurementImport, String> {

    MeasurementImport findByTimeOfCreation(Date timeOfCreation);
}

File: src/main/java/de/akquinet/ccsp/floodwatch/repository/MeasurementImportRepository.kt
package de.akquinet.ccsp.floodwatch.repository

import de.akquinet.ccsp.floodwatch.datamodel.MeasurementImport
import org.springframework.data.mongodb.repository.MongoRepository
import java.util.*

interface MeasurementImportRepository : MongoRepository<MeasurementImport, String> {

    fun findByTimeOfCreation(timeOfCreation: Date): MeasurementImport
}
